apps/api/src/view-models/resourceDescriptions/buildAllAggregateDescriptions.ts

```diff
@@ -94,5 +95,6 @@ export const buildAllAggregateDescriptions = (): Omit<AggregateInfo, 'link'>[] =
         type: resourceType,
         description,
         label: formatResourceType(resourceType),
+        pluralLabel: getPluralLabelForAggregateType(resourceType),
```

Comment:
Will be nice to do this dynamically with:
```ts
@Resource({
     type: 'vocabularyList',
     label: 'Vocabulary List',
     pluralLabel: 'Vocabulary Lists'
})
```

Reply:
Gotcha, cool!